src/_data/whats-new.yml

```diff
@@ -4,8 +4,88 @@ description:
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Tue Aug 18 11:56:55 2020
+updated: Mon Aug 31 15:26:14 2020
 entries:
+- description: Updated the Elastiicsearch version compatibility and software version
+    compatibility matrices to show Elasticsearch 7.7.x as the supported and recommended
+    version for Magento Commerce on the Cloud platform.
+  versions: 2.x
+  type: Technical
```

Comment:
We only add Major Updates and New Topics to whatsnew. Technical updates should be removed.

Reply:
@dobooth That's incorrect. Technical updates can have `whatsnew` statements, at the writer's discretion. Note the syntax of the `.whatsup.yml` file

```
labels:
  required:
    - New Topic
    - Major Update
  optional:
    - Technical
```

@meker12 Added a whatsnew to her PR, and that's a valid decision.